scripts/config-manual.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+S1="iface eth0 inet manual"
+
+#Remove existing settings for all other configurations from /network/interfaces
+# e.g. sed -i -e '/pattern/,+#d', deletes any line containing the pattern and # number of lines after that.
+sudo sed -i -e '/iface eth0 inet manual/,+1d' /etc/network/interfaces
```

Comment:
This sort of hard coding concerns me. what if there is more/less than this number of lines in the configuration? It might work right now on our current images, but this is not portable. I think we should have three static files inside of this repo, and copy them over the system file as necessary. This will make sure it goes off right.

Reply:
Look at usage of /etc/network/interfaces.d

Files in this directory are sourced by the main interfaces file. We should keep the eth0 configuration separate here. We can have three files with the total content configuration for eth0, just write over that file for the configuration step rather than all this `sed` business.

Reply:
Okay i think that would be way better. you are right.

Reply:
To further improve things, it looks like [this](https://unix.stackexchange.com/a/316051) is the correct way to do what we want. We edit the file once (when we update) to define our configurations, then load the one we want with `ifup/ifdown`.

Reply:
Now in our scripts, we can just turn the respective interface on in our script as its already included in the interfaces file.